webapp/src/mixins/RouteWatcher.vue

```diff
@@ -0,0 +1,13 @@
+<script>
+
+export default {
+    name: 'RouteWatcher',   
+    watch: {
+    '$route': function(n, o) {
+      console.log("test")
```

Comment:
probably remove this debug line? @sengineer0

Reply:
@newgene  It's removed. I also fixed an error in quick index page. It does not refresh data when the datasource changed by changing url